compiler/rustc_codegen_cranelift/example/alloc_example.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(start, box_syntax, core_intrinsics, alloc_prelude, alloc_error_handler)]
+#![feature(start, core_intrinsics, alloc_prelude, alloc_error_handler)]
```

Comment:
These actually do still use the box syntax. I guess I will need to update them at the next sync.

Reply:
Oh sorry about that. CI isn't ran for these so it didn't detect it but I could have double checked.